Before: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
After: b142e882d24c6799bdfa19908631dc83d6de0947f65b63a0d8c370645089848e
Patch: 3fea15483cbc500a42f707d1f8aeb4b9ee72d117b6147fbf2ae104fb4a839e3d
--- /dev/null	1980-01-01 00:00:00.000000000 +1000
+++ %name%.csv.tt	2012-09-13 14:57:39.000000000 +1000
@@ -0,0 +1,2 @@
+NAME,AGE
+<%= name %>,<%= age %>
--- /dev/null	1980-01-01 00:00:00.000000000 +1000
+++ %name%.txt.tt	2012-09-13 14:57:39.000000000 +1000
@@ -0,0 +1,4 @@
+My name is <%= name %>!
+I'm <%= age %> years old.
+Project name <%= project_name %>
+Bye.
--- /dev/null	1980-01-01 00:00:00.000000000 +1000
+++ corvid-features/template2.rb	2012-09-13 14:57:39.000000000 +1000
@@ -0,0 +1,7 @@
+install {
+  template2 '%name%.txt.tt'
+  template2 '%name%.csv.tt'
+}
+
+def name; 'George' end
+def age; 23 end
